Tests/Functional/Domain/Service/TranslationFileServiceTest.php

```diff
@@ -84,8 +83,7 @@ public function setUp()
             mkdir($this->l10nItFolder, 0777, true);
         }
 
-        $objectManager = GeneralUtility::makeInstance(ObjectManager::class);
-        $this->translationFileService = $objectManager->get(TranslationFileService::class);
+        $this->translationFileService = GeneralUtility::makeInstance(TranslationFileService::class);
```

Comment:
constructor arguments required?

Reply:
Will take care of the tests in a different patch.